internal/abi/fastly/types.go

```diff
@@ -106,8 +110,10 @@ func (s FastlyStatus) String() string {
 		return "HTTPHeadTooLarge"
 	case FastlyStatusHTTPInvalidStatus:
 		return "HTTPInvalidStatus"
+	case FastlyStatusLimitExceeded:
+		return "LimitExceeded"
 	default:
-		return "unknown"
+		return fmt.Sprintf("unknown error code: %d", s)
```

Comment:
I think something like
```
               return "FastlyStatus(" + strconv.Itoa(int(s)) + ")"
```
is a bit more Go like.  (Although note you'll need to add `strconv` as an import)